README.md

```diff
@@ -54,15 +54,15 @@ Craftable uses all the packages above. It also uses some other 3rd party package
 ## Requirements ##
 
 Craftable requires:
-- PHP 7.2+
+- PHP 7.4+
```

Comment:
Why is 7.4 required? Is not 7.3 enough as for Laravel 8?

Reply:
Yes, but Spatie's medialibrary requires 7.4